Assets/_BForBoss/_Entities/Scripts/LifeCycle.cs

```diff
@@ -3,7 +3,15 @@
 
 namespace Perigon.Entities
 {
-    public class LifeCycle
+    public interface ILifeCycle
+    {
+        public float MaxHealth { get; }
+        public float CurrentHealth { get; }
+        public bool Alive { get; }
```

Comment:
these don't have to be public

Reply:
and should be `isAlive`

Reply:
I'll fix the isAlive, but don't they need to be public? I'm using the floats in a different class

Reply:
no, because the class that implements this has to make them public anyway.

Reply:
Oh I see what you're saying, thats my bad